aeron-driver/src/main/c/media/aeron_udp_destination_tracker.c

```diff
@@ -109,7 +109,7 @@ int aeron_udp_destination_tracker_send(
     int result = (int)iov_length;
     int to_be_removed = 0;
 
-    *bytes_sent = 0;
+    *bytes_sent = iov->iov_len;
```

Comment:
This might not work correctly, because `send_func` might reset the value back to zero in some cases. See [aeron_udp_channel_transport_send](https://github.com/aeron-io/aeron/blob/512fb2464b2de3ba03b71dd1734d29b457e67371/aeron-driver/src/main/c/media/aeron_udp_channel_transport.c#L630) and the functions it calls.

Reply:
I think that's ok... because I think that would indicate an actual short send.

Reply:
This will over-count `bytes sent` as `bytes_sent` is also incremented in the `send_func`.
Thinking about it a bit more why is the fix necessary at all given the `send_func` logic? Could it be that the bug is actually in the `send_func` implementations?